substrate/frame/support/test/tests/construct_runtime_ui.rs

```diff
@@ -27,6 +27,9 @@ fn ui() {
 	// As trybuild is using `cargo check`, we don't need the real WASM binaries.
 	std::env::set_var("SKIP_WASM_BUILD", "1");
 
+	// Deny all warnings since we emit warnings as part of a Runtime's UI.
+	std::env::set_var("RUSTFLAGS", "--deny warnings");
+
```

Comment:
We already do this in the CI job?

Reply:
Yes but then the result is different from running it locally.